test/steps/offline_steps.go

```diff
@@ -0,0 +1,31 @@
+package steps
+
+import (
+	"fmt"
+
+	"github.com/cucumber/godog"
+)
+
+// OfflineSteps defines Cucumber step implementations around offline functionality.
+func OfflineSteps(suite *godog.Suite, fs *FeatureState) {
+	suite.Step(`^offline mode is disabled$`, func() error {
+		offline, err := fs.activeScenarioState.gitEnvironment.DeveloperRepo.IsOffline()
+		if err != nil {
+			return err
+		}
+		if offline {
+			return fmt.Errorf("expected to not be offline but am")
+		}
+		return nil
+	})
+	suite.Step(`^offline mode is enabled$`, func() error {
```

Comment:
These steps could probably be joined to reduce the code a bit

Reply:
I agree. As a matter of fact, I want to join all steps together when done. If we keep them organized alphabetically, it's relatively quick to locate steps even in a larger list of files. The schema to name files works only around 80% of the time.